terraform/lib/dependabot/terraform/file_updater.rb

```diff
@@ -149,7 +149,13 @@ def run_terraform_init
           output = e.message
 
           if output.match?(PRIVATE_MODULE_ERROR)
-            raise PrivateSourceAuthenticationFailure, output.match(PRIVATE_MODULE_ERROR).named_captures.fetch("repo")
+            repo = output.match(PRIVATE_MODULE_ERROR).named_captures.fetch("repo")
+            git_https_prefix = %r{^git::https://}
```

Comment:
I guess this could be a constant 🤷

Reply:
👍 Went ahead and did this. I was debating this since moving the definition up to the top of the file could hurt readability. That does seem to be a more consistent practice in this repo though so I went with it.